ShopifySharp/Services/InventoryLevel/InventoryLevelService.cs

```diff
@@ -33,5 +33,53 @@ public class InventoryLevelService : ShopifyService
 
             return await ExecuteRequestAsync<List<InventoryLevel>>(req, HttpMethod.Get, rootElement: "inventory_levels");
         }
+
+        /// <summary>
+        /// Deletes inventory for an item at specified location.  All items must keep inventory at at least one location.
+        /// </summary>
+        /// <param name="inventoryItemId">The ID of the inventory item.</param>
+        /// <param name="locationId">The ID of the location that the inventory level belongs to.</param>
+        public virtual async Task DeleteAsync(long inventoryItemId, long locationId)
+        {
+            await ExecuteRequestAsync(PrepareRequest($"inventory_levels.json?inventory_item_id={inventoryItemId}&location_id={locationId}"), HttpMethod.Delete);
+        }
+
+        /// <summary>
+        /// Updates the given <see cref="InventoryLevel"/>.
+        /// </summary>
+        /// <param name="updatedInventoryLevel">The updated <see cref="InventoryLevel"/></param>
+        /// <param name="disconnectIfNecessary">Whether inventory for any previously connected locations will be set to 0 and the locations disconnected. This property is ignored when no fulfillment service is involved.</param>
+        /// <returns>The updated <see cref="InventoryLevel"/></returns>
+        public virtual async Task<InventoryLevel> UpdateAsync(InventoryLevel updatedInventoryLevel, bool disconnectIfNecessary = false)
```

Comment:
Looking at the API docs for inventory levels, I think it would be better to rename this to method `SetAsync` since it's using a POST call and there are other endpoints (adjust, connect) that also technically update the inventory level.